_docs/app-metrics.md

```diff
@@ -1,5 +1,57 @@
-# App Metrics
 
+# Status usage data
+Starting release 1.14, the Status mobile app asks to share privacy-preserving, end-to-end encrypted data about how you use Status. Data is only ever shared if you opt in to doing so, you can review all data before it is sent and it is shared over Status' peer-to-peer network (Waku), just like a 1:1 message.
+
+Sharing data is strictly opt-in, and can be changed at any times in Settings in the app.
+
+### What is shared (Opt-in)
+Status is an open-source platform. To verify that the app only collects and shares the events and metadata listed below, you can view the rules set to store metrics data in the [source code](https://github.com/status-im/status-go/blob/develop/appmetrics/validators.go).
+
+- Your interactions with the app like clicks and screenviews
+- Background activity and internal processes
+- Settings and preferences
+
+In detail this means the app can collect and share:
+- Navigation events
+- Screens that are navigated to
+- Action events (for example button clicks, list swipes, switch toggles, etc)
+- The time events are created
+- Operating System
+- App version
+- Batch ID
+- Time since last session
+- Time since last data message sent
+
+For any data to be collected and shared, it needs to meet the rules set in the validator script. By policy of Status' Core Contributors, rules set in the validator script need to be assessed as posing a non-existent to low threat by being in:
+- local storage over time **(Specify what worst case can be and when this occurs)**
+- aggregated storage from Status over time
+
+### What will never be shared
+No data will be shared unless you opt-in. Furthermore, Status commits to never collect identifiable data in its broadest sense. This means that we will never collect anything that we believe can be linked back to you. Including but not limited to:
+- IP addresses
+- Random name
+- Chat key
+- Public Ethereum addresses
+- Account balance(s)
+- Account history
+- ENS name
+- Input field entries (browser address bar, chat key/ENS entry)
+- Any content you generate (images, messages, profile picture)
+- Contacts or other (favorite) lists (wallet and browser favorites)
+- Chat, group, community memberships
+
+While we employ a ‘can’t be evil’ approach. We highly recommend that you employ a ‘don’t trust, but verify’ approach. In this case, the above data can technically be logged, stored and shared. In order to verify that this does not happen, view the rules defined to store metrics data in the [source code](https://github.com/status-im/status-go/blob/develop/appmetrics/validators.go).
+
+### Purpose of the data
+Aggregated data is used to inform product development. Given our principles, the type of data that is collected and its rudimentary nature, there is no incentive for Status or other parties to use the data for any other purpose.
```

Comment:
Even though we feel there is no incentives to use it for profit, we should also explicitly state that we will never do so. Many data platforms didn't realize they were data platform until they had the treasure in hand, which then pushed a change to make profit from it.

Reply:
Good point. Added: "Status will never use these data for profit."